src/video-grid/VideoTileContainer.tsx

```diff
@@ -72,7 +72,7 @@ export function VideoTileContainer({
     audioContext,
     audioDestination,
     localVolume,
-    isLocal || maximised
+    isLocal
```

Comment:
I don't think I see why this got removed?

Reply:
Yeah, working out why it was there in the first place was also rather subtle. We routed audio differently when a video was maximised, using the AudioContainer things. This is no longer necessary since the audio elements can just stay in the DOM.

Reply:
Well, I don't think we would want to play audio both from `AudioSink`s and the spatial audio thingy when full-screened? What am I missing?

Reply:
Yep, the spatial audio pathways are only active when spatial audio is turned on. Previously we used a separate audio context based audio path when in fullscreen because all the other video elements went away. Now we use the audio elements whether fullscreen or not.

Reply:
Sorry, I think I am still missing something... - what about the case where you have spatial audio and are in fullscreen, how is audio routed in this case?

Reply:
Ah yes sorry, turns out I probably did have double audio here for the maximised video as it does go via the audio elements when maximised, spatial audio or no spatial audio. Fixed.